orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/MonitoredDeployStrategy.groovy

```diff
@@ -404,8 +404,13 @@ class MonitoredDeployStrategy implements Strategy {
       internalStageData.cloudProvider = baseContext.cloudProvider
       internalStageData.region = baseContext.region
       internalStageData.oldServerGroup = source?.serverGroupName
-      internalStageData.newServerGroup = stageData.deployServerGroups[baseContext.region].first()
-      internalStageData.parameters = stageData.deploymentMonitor.parameters
+      //TODO (gyardeni): check if it is true here?
```

Comment:
nit: fixing whitespace to `TODO(gyardeni):` may make things a little simpler to search for in the future. You should also consider being clearer about "it", rather than assuming the reader knows what "it" means (e.g. me! :).

Reply:
Thanks @robzienert, will remember it for next time!